assets/erc-7579/IMSA.sol

```diff
@@ -46,17 +31,6 @@ interface IERC7579Account {
         external
         returns (bytes[] memory returnData);
 
-    /**
-     * @dev ERC-4337 validateUserOp according to ERC-4337 v0.7
-     *         This function is intended to be called by ERC-4337 EntryPoint.sol
-     * this validation function should decode / sload the validator module to validate the userOp
-     * and call it.
-     * @param userOp PackedUserOperation struct (see ERC-4337 v0.7+)
-     */
-    function validateUserOp(PackedUserOperation calldata userOp, bytes32 userOpHash, uint256 missingAccountFunds)
```

Comment:
isn't this a bit unintuitive if you're building an account for 4337?

Reply:
Imo its fine because this function is in the ERC-4337 interface. So if you're building a 4337 account, you'll check out that standard + interface too

Reply:
> Imo its fine 
I agree. Going with validateUserOp is intuitive if you want to support 4337